### DISCUSSION TITLE
WebSockets support

### DISCUSSION THREAD

**Discussion by fad3t** [apache/apisix#11186] [Q&A]
Hello all,

I'm trying to get WebSockets working using APISIX.
This is working as expected using an `ApisixRoute` resource, but not using an `Ingress`. I'm using the annotation found in the documentation: `k8s.apisix.apache.org/enable-websocket: "true"`.

Is this still supported?
Thx.

**Comment by shuaiyy** [comment]
I tried in 3.8.0, works fine.
